backend/pytype.config

```diff
@@ -0,0 +1,78 @@
+# NOTE: All relative paths are relative to the location of this file.
+
+[pytype]
+
+# Space-separated list of files or directories to exclude.
+exclude =
+    .idea
+    .mypy_cache
+    .pytype
+
+# Space-separated list of files or directories to process.
+inputs =
+    .
+
+# Keep going past errors to analyze as many files as possible.
+keep_going = False
+
+# Run N jobs in parallel. When 'auto' is used, this will be equivalent to the
+# number of CPUs on the host system.
+jobs = 4
```

Comment:
This is mostly a copy pasta, right? Can't parameters like `jobs` be removed from this config?